R/rows-db.R

```diff
@@ -62,9 +57,13 @@ NULL
 #' @export
 #' @rdname rows-db
 rows_insert.tbl_dbi <- function(x, y, by = NULL, ...,
-                                in_place = NULL, copy = FALSE, check = NULL,
+                                in_place = FALSE,
+                                conflict = NULL,
+                                copy = FALSE,
                                 returning = NULL) {
 
+  stopifnot(identical(conflict, "ignore"))
+
```

Comment:
Should our doc explain that (and why) only "ignore" is supported for `conflict` and `unmatched` ?

Reply:
This is only compatibility code that will be removed as soon as dbplyr 2.2.0 hits CRAN. No need to spend time here, I guess.